config/initializers/require_dotenv_in_dev.rb

```diff
@@ -0,0 +1 @@
+require 'dotenv' if Rails.env.development?
```

Comment:
I'd like to see a comment about this. Why are we doing it this way? Some documentation would be helpful.